### PR TITLE
Last changes to reviewed smartdocs

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: references/virtual-disk-cache-modes.xml [hunk 1]
```diff
@@ -149,16 +155,23 @@
       factors. If the guest storage is hosted on a clustered file system, is read-only, or is
       marked shareable, then the cache mode is ignored when determining if migration can be
       allowed. Otherwise, &libvirt; does not allow migration unless the cache mode is set to
-      <literal>none</literal>. However, this restriction can be overridden with the
-      <option>--unsafe</option> option to the migration APIs, which is also supported by
-      <command>virsh</command>. For example:
+      <literal>none</literal>.
     </para>
+    <warning>
```
Comment (janajaeger) [inline references/virtual-disk-cache-modes.xml:160]: Admonitions of any kind should always carry a title!

File: snippets/unsafe-cache-mode-warning.xml
Comment (janajaeger) [inline snippets/unsafe-cache-mode-warning.xml:1]: Please add a title. Every admonition should have one.

**Review by janajaeger — COMMENTED** [review]
Very nice :) Just add the missing titles to the warnings you added and be done with it all! Thanks very much!

File: references/systemd-securing-techniques.xml [hunk 2]
```diff
@@ -47,29 +48,30 @@
       <term>InaccessibleDirectories=/home</term>
       <listitem>
         <para>
-          This option makes the specified directories inaccessible to the service.
+          This option makes the specified directories inaccessible to the service. This option
+          narrows the range of directories that can be read or modified by the service, for
+          example, to secure user's private files.
```
Comment (dariavladykina) [inline references/systemd-securing-techniques.xml:53]:
```suggestion
          example, to secure users' private files.
```

File: tasks/systemd-analyze-security-service-files.xml [hunk 3]
```diff
@@ -25,10 +25,11 @@
   <para>
     Use the <command>systemd-analyze security</command> command to analyze security settings of
     specified &systemd; service units. If unit names are not specified, the command inspects
-    security settings of all currently loaded service units. The command calculates an overall
-    <quote>exposure level</quote> which is an estimation in the range 0.0 to 10.0, showing how
-    exposed a service is security-wise. High values indicate little loose security provisions,
-    while low exposure levels indicate tight security restrictions.
+    security settings of all currently loaded service units. We recommend running the command after
+    a specific &systemd; unit file is updated. The command calculates an overall <quote>exposure
+    level</quote> which is an estimation in the range 0.0 to 10.0, showing how exposed a service is
```
Comment (dariavladykina) [inline tasks/systemd-analyze-security-service-files.xml:30]:
```suggestion
    level</quote> that is an estimation in the range 0.0 to 10.0, showing how exposed a service is
```

File: tasks/vm-assign-pci-device-troubleshooting.xml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+title Xen -- SUSE Linux Enterprise Desktop 15 SP5 - 6.1.0-0.6.6
+root (hd0,5)
+kernel /boot/xen.gz vga=mode-0x314 iommu=no-intremap
+module /boot/vmlinuz-6.1.0-0.6.6-xen root=/dev/sda6 resume=/dev/sda5 splash=silent showopts vga=0x314
+module /boot/initrd-6.1.0-0.6.6-xen</screen>
+  </section>
+  <section xml:id="vm-assign-pci-device-troubleshooting-device-missing">
+    <title>Virtual machine complains that the passthrough device is missing</title>
+    <para>
+      This often happens if you assign a removable host device&mdash;such as USB flash
```
Comment (dariavladykina) [inline tasks/vm-assign-pci-device-troubleshooting.xml:45]:
```suggestion
      This often happens if you assign a removable host device&mdash;such as a USB flash
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see a few minor comments, thanks.

**Pull request merged**